corehq/elastic.py

```diff
@@ -144,7 +179,7 @@ class ESError(Exception):
     pass
 
 
-def run_query(index_name, q, debug_host=None):
+def run_query(index_name, q, debug_host=None, es_instance=ES_DEFAULT_INSTANCE):
```

Comment:
Instead of passing in a string that looks in a dictionary to get this, why not pass in the actual instance?

That would allow you to have the export ES instance defined in corehq/apps/export and the default value could just be `None` which defaults to the old behavior of calling get_es_new